controllers/flowlogspipeline/metrics_definitions/top_pods_egress_bytes_total.yaml

```diff
@@ -0,0 +1,29 @@
+#flp_confgen
+description:
+  This metric observes the outgoing traffic
+details:
+  Sum bytes for outgoing traffic per source pod / service
+usage:
+  Evaluate network outgoing usage breakdown per pod / service
+tags:
+  - egress
+  - bytes
+  - pods
+  - services
+encode:
+  type: prom
+  prom:
+    metrics:
+      - name: top_pods_egress_bytes_total
+        type: counter
+        valuekey: Bytes
+        filters:
+        - key: FlowDirection
+          value: "1"
+        - key: Duplicate
+          value: "false"
+        labels:
+          - SrcK8S_Name
```

Comment:
My understanding was that using SrcK8S_Name as label would create too much granularity for prometheus.

May be we should document than enabling this metric can create a lot of pressure on prometheus?

Reply:
my take on that is that Src alone at pod granularity is ok (cardinality is N - number of pods), Dst alone is ok (cardinality is still N), but Src + Dst is too much (cardinality is N²)

After all there's already many metrics that are per pod, that would be just another one.

(I haven't looked yet at this `ExtractTimeBased`)

Reply:
I'm testing some traffic generation on services and noticed that I see the generated traffic only using Reporter Node Source + Filter on Destination Resource Service

![image](https://github.com/netobserv/network-observability-operator/assets/91894519/4060c54d-13a3-4e22-abdb-a7d29df5e0b5)
![image](https://github.com/netobserv/network-observability-operator/assets/91894519/5931ab23-370d-4b4e-8341-63ee9c843394)

So maybe at the end we need only one metric `top_pods_bytes_total` containing only destinations fields and mixing both FlowDirection (egress / ingress) ? WDYT ?

Reply:
You can directly try https://github.com/netobserv/network-observability-operator/pull/392 to get the dashboards :wink: